build_and_test.sh

```diff
@@ -92,4 +92,18 @@ then
         echo "OpenFAM test with shared memory configuration failed. exit..."
         exit 1
 fi
+
```

Comment:
I am thinking if we should keep it as a separate script and not to run always we run build_and_test.sh. 
What do you think @faizan-barmawer.